2. Xizmatlar/2.4 Aggregatsiyalar/2.4 Aggregatsiyalar.md

```diff
@@ -0,0 +1,17 @@
+<h1>2.4 Aggregatsiya(To'plash) Xizmatlari </h1>
+<h3>2.4.0 Kirish</h3>
+Aggregatsiya xizmatining asosiy vazifasi, biznes mantiq qatlami va tashqi qatlamlari orasida bitta bog'liqlikni ko'rsatishdir. Bu bir nechta xizmatlarning har qanday o'zgarishlarining, bir xil shartnoma asosida yig'ilishi va bir ko'rsatuvchi tarkibiy qismga, bir mantiqiy qatlam orqali ko'rsatilishini ta'minlashdir.
+
+Aggregatsiya xizmatlari o'zida hech qanday biznes mantiqni saqlamaydi. Ular shunchaki har qanday sonli xizmatlarni birlashtiradigan to'qig'ichdir. Ular xizmatlar qatlami bilan ishlovchi bo'lishlari mumkin va odatda ular ushbu xizmatlarni chaqirishni ko'rsatadi. Bu yerda aggregatsiya xizmatining kodi misolini ko'rish mumkun:
```

Comment:
to'qig'ichdir => tushuna olmadim bu so'zni

Reply:
Changed the word, to'qig'ich - jipslashtirgich